src/zenml/cli/stack_recipes.py

```diff
@@ -0,0 +1,1013 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+
+#       https://www.apache.org/licenses/LICENSE-2.0
+
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Functionality to handle downloading ZenML stacks via the CLI."""
+
+import os
+import shutil
+import subprocess
+import sys
+from pathlib import Path
+from typing import Any, List, Optional, cast
+
+import click
+import python_terraform  # type: ignore
+from packaging.version import Version, parse
+from rich.markdown import Markdown
+from rich.text import Text
+
+from zenml.cli import utils as cli_utils
+from zenml.cli.cli import TagGroup
+from zenml.cli.stack import import_stack, stack
+from zenml.console import console
+from zenml.exceptions import GitNotFoundError
+from zenml.io import fileio
+from zenml.logger import get_logger
+from zenml.utils import io_utils
+from zenml.utils.analytics_utils import AnalyticsEvent, track_event
+
+logger = get_logger(__name__)
+
+EXCLUDED_RECIPE_DIRS = [""]
+STACK_RECIPES_GITHUB_REPO = "https://github.com/zenml-io/mlops-stacks.git"
+STACK_RECIPES_REPO_DIR = "zenml_stack_recipes"
+VARIABLES_FILE = "values.tfvars.json"
+ALPHA_MESSAGE = (
```

Comment:
I would even add here that they should go to the `STACK_RECIPES_GITHUB_REPO` to report issues

Reply:
Agreed